MAUI/AIAssistView/header.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: Header in .NET MAUI AI Assist View control | Syncfusion
+description: Learn here all about Header support in Syncfusion .NET MAUI AI Assist View (SfAIAssistView) control and more.
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Header in .NET MAUI AI Assist View (SfAIAssistView)
+
+This section explains how to define and customize the header in the `SfAIAssistView`.
+
+## Show Header
+
+The `SfAIAssistView` control allows you to display a default header by configuring the `ShowHeader` property. When this property is set to `true`, the default header will be shown at the top of the assist view. The default value of the `ShowHeader` property is `false`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+         <sfAIAssistView:SfAIAssistView x:Name="sfAIAssistView"
+                                        ShowHeader="True"/>  
+
+{% highlight c# hl_lines="6" %} 
+
+    SfAIAssistView sfAIAssistView; 
+    public MainPage() 
+    { 
+        InitializeComponent(); 
+        this.sfAIAssistView = new SfAIAssistView();
+        this.sfAIAssistView.ShowHeader = true;
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Header text
+
+The `SfAIAssistView` control allows you to customize the header text using the `HeaderText` property. The default value of the HeaderText is `How can I help you?`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+         <sfAIAssistView:SfAIAssistView x:Name="sfAIAssistView"
+                                        HeaderText="Ask AI"
+                                        ShowHeader="True"/>  
+
+{% endhighlight %} 
+
+{% highlight c# hl_lines="6" %} 
+
+    SfAIAssistView sfAIAssistView; 
+    public MainPage() 
+    { 
+        InitializeComponent(); 
+        this.sfAIAssistView = new SfAIAssistView();
+        this.sfAIAssistView.HeaderText = "Ask AI";
+        this.sfAIAssistView.ShowHeader = true;
+        this.Content = sfAIAssistView; 
+     } 
+
+{% endhighlight %}
+{% endtabs %}
+
+## Header customization
+
+The `SfAIAssistView` control allows you to fully customize the header's appearance by using the `HeaderTemplate` property. This property lets you define a custom layout and style for the header.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="46" %}
```

Comment:
seems highlight line index is not correct.

Reply:
Addressed review comments.